src/cbwd/main.cc

```diff
@@ -187,10 +196,7 @@ int main(int argc, char** argv) {
     configuration_parser parser;
     config = parser.parse(config_filename);
   } catch (std::exception const& e) {
-    std::cout << "ERROR: could not parse the configuration file '"
-              << config_filename << "': " << e.what() << '\n';
-    logging::error(logging::medium)
-        << "watchdog: could not parse the new configuration: " << e.what();
+    logger->error("ERROR: could not parse the configuration file '{}': {} '\n'watchdog: Could not parse the configuration: {}",config_filename, e.what(), e.what());
```

Comment:
I did not see this previously. You write almost twice the same message. Could you make a simple message with the config_filename and the e.what(). No need to write them twice !